app/jobs/upvs/fetch_services_with_forms_list_job.rb

```diff
@@ -3,7 +3,7 @@ class Upvs::FetchServicesWithFormsListJob < ApplicationJob
 
   def perform(url, downloader: HarvesterUtils::Downloader)
     csv_file = downloader.download_file(url)
-    csv_options = { col_sep: File.open(csv_file) { |f| f.readline }.include?(';') ? ';' : ',', headers: true }
+    csv_options = { col_sep: File.open(csv_file) { |f| f.readline }.include?(';') ? ';' : ',', headers: true, encoding: "windows-1250" }
```

Comment:
https://rubystyle.guide/#consistent-string-literals-single-quote

```suggestion
    csv_options = { encoding: 'windows-1250', col_sep: File.open(csv_file) { |f| f.readline }.include?(';') ? ';' : ',', headers: true }
```